proto/neutron/interchaintxs/v1/tx.proto

```diff
@@ -26,6 +27,10 @@ message MsgRegisterInterchainAccount {
   string connection_id = 2 [ (gogoproto.moretags) = "yaml:\"connection_id\"" ];
   string interchain_account_id = 3
       [ (gogoproto.moretags) = "yaml:\"interchain_account_id\"" ];
+  repeated cosmos.base.v1beta1.Coin register_fee = 4 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
```

Comment:
ah, I remember that we discussed an advantage of having such a parameter: if the fee increases one day, the smart contracts with the previously set fee will stop working thus making their owners to increase the fee manually thus forcing the owners to, so to say, acknowledge the change. otherwise a significant change of this parameter could bring an unexpected and implicit operational costs change.

whereas I was the one who arose this point, now I don't think it makes sense: this so to say acknowledgement will bring huge operational load on protocols running their contracts on neutron and particularly for the DAO-controlled ones: they will have to pass a `update_contract_params` call through governance process to increase the fee even if it changed slightly. I think that making a simple force charge without requiring contracts to pass the value explicitly is totally ok: if this parameter ever changes significantly, I think all the protocol owners will become aware of it because such a change will be only possible via a proposal on Neutron